api/controllers/DefaultController.js

```diff
@@ -26,7 +26,7 @@ module.exports = class DefaultController extends Controller {
    * @param res
    */
   default(req, res) {
-    let lang = req.acceptsLanguages('en', 'en-US', 'en-UK', 'fr', 'fr-FR')
+    let lang = req.acceptsLanguages('en', 'en-US', 'en-UK', 'fr', 'fr-FR','ru','ru-RU')
```

Comment:
This will not work as the UI part (angular 5 app) is not translated into Russian, this code is only to take front files under the correct language, ui shouldn't work with this